src/actions/ScanActions.js

```diff
@@ -194,6 +194,13 @@ export const parseScannedUri = (data: string, customErrorTitle?: string, customE
       }
     ]
 
+    if (fioAddress) {
```

Comment:
No more implicit null checks in the GUI. Only booleans should be used this way and other types should be null-checked explicitly. I'm only adding this single comment but this appears in a few places across the PR.

Reply:
yes, sure, my bad, it was just moved from other place and I did not rewrite it from scratch